graph/src/lib.rs

```diff
@@ -61,7 +61,7 @@ pub mod prelude {
     pub use tokio;
     pub use web3;
 
-    pub type DynTryFuture<'a, Ok, Err = Error> =
+    pub type DynTryFuture<'a, Ok = (), Err = Error> =
```

Comment:
Doesn't this make `DynTryFuture` impossible to use in situations where the result value is not `()`?

Reply:
This is just a default. So, you can write `DynTryFuture<'a, u32>` (defaults to Error = failure::Error), `DynTryFuture<'a, u32, MyErrorType>` (explicitly specify everything) or just `DynTryFuture<'a>`.